AdsLib/AdsDef.cpp

```diff
@@ -62,6 +62,54 @@ AmsNetId::AmsNetId(uint8_t id_0, uint8_t id_1, uint8_t id_2, uint8_t id_3, uint8
 }
 
 AmsNetId::AmsNetId(const std::string& addr)
+{
+	assign(addr);
+}
+
+AmsNetId::AmsNetId(const AmsNetId& rhs)
+{
+	assign(rhs);
+}
+
+
+bool AmsNetId::operator == (const AmsNetId& rhs) const

```

Comment:
```suggestion
bool AmsNetId::operator==(const AmsNetId& rhs) const
```
... also for the other operators